src/overview/rlc.md

```diff
@@ -0,0 +1,130 @@
+---
+title: RLC Token
+description:
+  The RLC token is the native cryptocurrency that powers the iExec decentralized
+  computing ecosystem
+---
+
+<div class="flex flex-col items-center mb-8">
+  <img :src="rlcGif" alt="RLC Token Animation" class="w-80 h-80 mb-0" />
+  <h1 class="text-3xl font-bold text-center mb-2">RLC Token</h1>
+  <p class="text-lg text-center text-gray-600 max-w-2xl">The native cryptocurrency that powers the entire iExec decentralized confidential computing ecosystem</p>
+</div>
+
+**RLC** (Run on Lots of Computers) serves as the primary medium of exchange for
+all interactions within the protocol, enabling users to access confidential
+computing services and rewarding providers for their contributions.
+
+## 🎯 What RLC Enables
+
+RLC is essential for interacting with the iExec protocol and accessing its
+decentralized confidential computing services:
```

Comment:
Either close the sentence here or continue with bullet points or something else. The current formatting seems like both sections are separate.

<img width="950" height="405" alt="image" src="https://github.com/user-attachments/assets/104cd5fa-d12d-4f37-882d-caea153ab4c5" />